docs/user/architecture.rst

```diff
@@ -0,0 +1,83 @@
+Architecture Overview
+=====================
+
+For scalability, GeoMesa is built on technologies that can use Apache Hadoop. This includes data stores such as Accumulo, HBase, and Google Cloud Bigtable, as well as the Apache Kafka message broker for streaming data. Apache Storm lets you define information sources and manipulations to allow batch distributed processing of streaming data with GeoMesa, and a GeoMesa environment can also take advantage of Apache Spark to do large-scale analytics of stored and streaming data.
+
+.. image:: _static/img/GMHadoopInfrastructure.png
+   :align: center
+
+
+.. _geomesa_and_geoserver:
+
+GeoMesa and GeoServer
+---------------------
+
+GeoMesa's GeoServer plugins let GeoServer work with data stored using GeoMesa tables. The plugin maps the GeoServer capabilities associated with data sources to the GeoMesa formatted tables and provides a number of optimizations to support specific use cases.
+
+If an application already uses GeoServer, integration with GeoMesa is simply a matter of adding a new datastore to GeoServer and updating the application’s configuration.
+
+.. image:: _static/img/GeoServer-Integration.png
+   :scale: 75%
+   :align: center
+
+
+Integration with GeoMesa
+------------------------
+
+To expose the geospatial data that it stores for use by your applications,
+GeoMesa implements `GeoTools <http://geotools.org/>`_ interfaces to provide HTTP access to the following Open Geospatial Consortium standards:
+
+* `Web Feature Service (WFS) <http://www.opengeospatial.org/standards/wfs>`_
+* `Web Mapping Service (WMS) <http://www.opengeospatial.org/standards/wms>`_
+* `Web Processing Service (WPS) <http://www.opengeospatial.org/standards/wps>`_
+* `Web Coverage Service (WCS) <http://www.opengeospatial.org/standards/wcs>`_
+
+For ingest, we recommend using Apache Storm. The following diagram shows one possible ingest architecture:
```

Comment:
I don't think we recommend using storm... i would change this to say something like 'we provide streaming and batch ingestion' or some such.